attestation/context.go

```diff
@@ -171,6 +183,8 @@ func (ctx *AttestationContext) runAttestor(attestor Attestor) {
 	if producer, ok := attestor.(Producer); ok {
 		ctx.addProducts(producer)
 	}
+
+	log.Debugf("Finished %v attestor...", attestor.Name())
```

Comment:
Thinking of making this a `Infof` instead of `Debugf` so you get more understandable output.